data_structures/src/transaction.rs

```diff
@@ -182,7 +182,7 @@ pub struct CommitTransactionBody {
     // TODO: Discussion about collateral
     //pub collateral: Vec<Input>, // ValueTransferOutput
     pub dr_pointer: Hash, // DTTransaction hash
```

Comment:
This should also be a doc comment
```suggestion
    /// DRTransaction hash
    pub dr_pointer: Hash,
```